guide/02-api-overview/overview24.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p><em>renderer</em>&nbsp;property - returns&nbsp;<a href="https://github.com/python/api-reference/arcgis.map.toc.html#arcgis.map.renderers.RendererManager" target="_blank">RendererManager</a>&nbsp;object to manage symbology for layer entered as <strong>teh</strong>&nbsp;<em>index</em>&nbsp;argument.</p><p> </p><p>"teh" to "the"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2053/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>Fixed in Noah's PR!</p>